### PR TITLE
Fix dangling comma in labor category description

### PR DESCRIPTION
**Pull request by jseppi**
Use a regex to remove an ending comma and trailing whitespace from `laborCategory` in the `Description` component.

Closes https://github.com/18F/calc/issues/1236

### CODE REVIEW

**Comment by jseppi** [comment]
whoa, `bandit` is failing because of `mark_safe` in our various admin views.

**Comment by jseppi** [comment]
Weirdly, bandit is not failing locally. I confirmed I have the same version installed (`1.3.0`) so I'm not clear why it is failing in Travis and not locally.

**Comment by jseppi** [comment]
Same thing also caused a failure in recent PR merge to `develop`: https://travis-ci.org/18F/calc/builds/191372270. Plz review @toolness.

File: frontend/source/js/data-explorer/components/description.jsx [hunk 1]
```diff
@@ -30,7 +30,11 @@ export function Description({
   const filters = [];
 
   if (laborCategory) {
-    filters.push(<DescriptionFilter key="lab">{laborCategory}</DescriptionFilter>);
+    filters.push(
+      <DescriptionFilter key="lab">
+        {laborCategory.replace(/,\s*$/, '')}
```
Comment (toolness) [inline frontend/source/js/data-explorer/components/description.jsx]: Could you either add a comment / factor this out into a separate nicely-named function / put it into a nicely-named variable / etc to indicate that we're stripping the trailing comma out here?  For folks like me who find regexps hard to read... 😣

**Comment by toolness** [comment]
Noice! Plz merge immediately!

**Comment by toolness** [comment]
Post-haste!

**Comment by jseppi** [comment]
heh, now that I'm thinking about it some more, it _might_ be better if the comma-stripping logic happened in `LaborCategory`'s `setChange` method, because that dangling comma is heading to other places (like the url, for instance: `http://app.calc.docker/?q=Software%20Engineer%2C&min_experience=1&sort=-labor_category`).

ehhhhhhhh

**Comment by toolness** [comment]
Eh, let's save it for a different PR :)

**Pull request merged**